src/components/EmbeddedNavSection.vue

```diff
@@ -104,7 +104,7 @@
             {{ $t('header.feedback-nav-item') }}
           </NuxtLink>
           <a
-            href="https://api.creativecommons.engineering/"
+            href="https://api.creativecommons.engineering/v1/"
```

Comment:
```suggestion
            href="https://api.openverse.engineering/v1/"
```

The main goal here was to switch from `creativecommons` to `openverse` 😄